vendors/espressif/boards/ports/ota_pal_for_aws/ota_pal.h

```diff
@@ -23,33 +23,40 @@
  * http://www.FreeRTOS.org
  */
 
-#ifndef _AWS_OTA_PAL_H_
-#define _AWS_OTA_PAL_H_
+/**
+ * @file ota_pal.h
+ * @brief Function declarations for ota_ptal.c.
+ */
+
+#ifndef _OTA_PAL_H_
+#define _OTA_PAL_H_
```

Comment:
Seems like the ifndef/define in some files we are starting with "_" and some we are not. Can you make these consistent?

Reply:
Updated; Trailing underscore and removed the suffix.